### PR TITLE
Tech/connect

### PR DESCRIPTION
**Pull request by kevin-verschaeve**

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -78,3 +78,8 @@ services:
         class: Twig_Extensions_Extension_Text
         tags:
             - { name: twig.extension }
+
+sensio_labs_connect:
+    app_id: %sensiolabs_connect.app_id%
+    app_secret: %sensiolabs_connect.app_secret%
+    scope: %sensiolabs_connect.scope%
```
Comment (mmarchois) [inline app/config/config.yml]: CS

File: app/config/routing.yml [hunk 2]
```diff
@@ -3,5 +3,8 @@ sensio_labs_job_board:
     type:     annotation
     prefix:   /
 
-#_sensiolabs_connect:
-#    resource: "@SensioLabsConnectBundle/Resources/config/routing.xml"
+_sensiolabs_connect:
+    resource: "@SensioLabsConnectBundle/Resources/config/routing.xml"
+
+oauth_logout:
+    pattern: /session/logout
```
Comment (mmarchois) [inline app/config/routing.yml]: CS

File: src/SensioLabs/JobBoardBundle/Resources/views/layout.html.twig [hunk 3]
```diff
@@ -76,6 +77,7 @@
             {% javascripts output='js/common.js'
                 '@SensioLabsJobBoardBundle/Resources/assets/jquery.js'
                 '@SensioLabsJobBoardBundle/Resources/assets/common.js'
+                'http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js'
```
Comment (salahm) [inline src/SensioLabs/JobBoardBundle/Resources/views/layout.html.twig]: there is already jquery 2 lines above

File: src/SensioLabs/JobBoardBundle/Resources/views/layout.html.twig [hunk 4]
```diff
@@ -10,6 +10,7 @@
                 'bundles/sensiolabsjobboard/css/backend.css'
                 'bundles/sensiolabsjobboard/css/jquery/jquery-ui-1.9.2.custom.css'
                 'bundles/sensiolabsjobboard/css/custom.css'
+                'http://connect.sensiolabs.com/css/sln.css'
```
Comment (salahm) [inline src/SensioLabs/JobBoardBundle/Resources/views/layout.html.twig]: not sure this will work.

**Pull request closed without merging**